projects/demo/src/modules/app/app.component.ts

```diff
@@ -35,13 +35,15 @@ import {TUI_VERSION_MANAGER_PROVIDERS} from './version-manager/version-manager.p
         CustomHostComponent,
         TuiTableBarsHostModule,
         TuiTextfieldControllerModule,
+        TuiDestroyService,
```

Comment:
remove "TuiDestroyService" from imports array, and add import TuiDestroyService from @taiga-ui/cdk